db/connection.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+require('dotenv/config');
```

Comment:
You don't need to require `dotenv` everywhere, it's already called once in app.js